File: java/Calculator/src/Calculator.java
import java.util.Scanner;

public class Calculator {
public static void main(String[] args) {
	Scanner s=new Scanner(System.in);
	System.out.println("enter first number..");
	int num1=s.nextInt();
	System.out.println("enter second number..");
	int num2=s.nextInt();
	System.out.println("entered numbers are.. "+num1+num2);
	System.out.println(" 1.addition \n 2.substraction \n 3.multiplication \n 4.division \n");
	int value=s.nextInt();
	
	
	
switch(value) {
case 1:
	addition(num1,num2);
	break;
case 2:
	substraction(num1,num2);
	break;
case 3:
	multiplication(num1,num2);
	break;
case 4:
	division(num1,num2);
	break;
default:
	System.out.println("\n sorry try again..");
	break;
}



}

public static void addition(int num1,int num2) {
	System.out.println("\n addition : "+(num1+num2));
}

public static void substraction(int num1,int num2) {
	System.out.println("\n substraction : "+(num1-num2));
}
public static void multiplication(int num1,int num2) {
	System.out.println("\n multiplication : "+(num1*num2));
}
public static void division(int num1,int num2) {
System.out.println("\n division : "+(num1/num2));
}
}

File: week1/calculator.c
#include<stdio.h>
#include<stdlib.h>
int main()
{
    int n1,n2,result,choice;
    printf("enter two numbers\n");
    scanf("%d%d",&n1,&n2);
    printf("\n entered numbers are %d,%d \n---------------",n1,n2);
    printf("\n 1.addition \n 2.multiplication \n 3.substraction \n 4.division \n-----------------\n");
    scanf("%d",&choice);
    if(choice==1)
    {
        result=n1+n2;
    }
    else if(choice==2)
    {
        result=n1*n2;
    }else if(choice==3)
    {
        result=n1-n2;
    }else if (choice==4)
    {
        result=n1/n2;
    }
    else{
        printf("\ntry again");
    }
    printf("\nthe result=%d \n",result);
}